### PR TITLE
Psalm level 2

### PR DESCRIPTION
**Pull request by vjik**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ❌
| Breaks BC?    | ✔️

### CODE REVIEW

File: src/Command/DebugEventsCommand.php [hunk 1]
```diff
@@ -49,7 +50,6 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         if ($input->hasOption('groups') && $input->getOption('groups')) {
             $build = $this->getConfigBuild($config);
             $groups = array_keys($build);
-            ksort($groups);
```
Comment (xepozz) [inline src/Command/DebugEventsCommand.php:52]: For what reason?

File: src/Command/DebugContainerCommand.php [hunk 2]
```diff
@@ -116,7 +116,6 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         if ($input->hasOption('groups') && $input->getOption('groups')) {
             $build = $this->getConfigBuild($config);
             $groups = array_keys($build);
-            ksort($groups);
```
Comment (xepozz) [inline src/Command/DebugContainerCommand.php:119]: Same as previous

File: config/params.php [hunk 3]
```diff
@@ -62,7 +61,6 @@
             LoggerInterface::class => [LoggerInterfaceProxy::class, LogCollector::class],
             EventDispatcherInterface::class => [EventDispatcherInterfaceProxy::class, EventCollector::class],
             ClientInterface::class => [HttpClientInterfaceProxy::class, HttpClientCollector::class],
-            CacheInterface::class,
```
Comment (xepozz) [inline config/params.php:65]: Why?

**Review by xepozz — CHANGES_REQUESTED** [review]

File: src/Command/DebugEventsCommand.php (hunk 1 above)
Comment (vjik) [reply src/Command/DebugEventsCommand.php:52]: Not need. array_keys returns already sorted array.

File: config/params.php (hunk 3 above)
Comment (vjik) [reply config/params.php:65]: Seems, code don't work with integer keys.

File: src/Command/DebugEventsCommand.php (hunk 1 above)
Comment (xepozz) [reply src/Command/DebugEventsCommand.php:52]: https://3v4l.org/CmbjW

File: config/params.php (hunk 3 above)
Comment (xepozz) [reply config/params.php:65]: CacheInterface should be also proxied. Could you add it then?

File: src/Command/DebugEventsCommand.php (hunk 1 above)
Comment (vjik) [reply src/Command/DebugEventsCommand.php:52]: Ksort is sorted by keys

Comment (vjik) [reply src/Command/DebugEventsCommand.php:52]: Maybe here need to use sort instead ksort?

Comment (xepozz) [reply src/Command/DebugEventsCommand.php:52]: > Ksort is sorted by keys

What?

There should be a sorted list.

Comment (vjik) [reply src/Command/DebugEventsCommand.php:52]: See: https://3v4l.org/FAHBf

File: config/params.php (hunk 3 above)
Comment (vjik) [reply config/params.php:65]: Hm, proxy and collector for `CacheInterface` not exist.

File: src/Command/DebugEventsCommand.php (hunk 1 above)
Comment (xepozz) [reply src/Command/DebugEventsCommand.php:52]: Oh gotcha. You're right, there should be `sort` instead

Comment (vjik) [reply src/Command/DebugEventsCommand.php:52]: Added `sort`.

**Pull request merged**